rdflib/plugins/shared/jsonld/context.py

```diff
@@ -437,7 +441,9 @@ def _fetch_context(self, source, base, referenced_contexts):
         if source_url in self._context_cache:
             return self._context_cache[source_url]
 
-        source = source_to_json(source_url)
+        input_source = self.resolver.resolve(source_url, format="json-ld")
+        with contextlib.closing(input_source.getByteStream()) as stream:
+            source = json.load(stream)
```

Comment:
This will end poorly, resolve returns a tuple, this seems to expect an input_source